tests/integration/src/test/java/org/apache/pulsar/tests/integration/io/sources/debezium/DebeziumMsSqlSourceTester.java

```diff
@@ -0,0 +1,165 @@
+        sourceConfig.put("snapshot.mode", "schema_only");
+        sourceConfig.put("database.history.pulsar.service.url", pulsarServiceUrl);
+        sourceConfig.put("topic.namespace", "debezium/mssql");
+    }
+
+    @Override
+    public void setServiceContainer(DebeziumMsSqlContainer container) {
+        log.info("start debezium MS SQL server container.");
+        Preconditions.checkState(debeziumMsSqlContainer == null);
+        debeziumMsSqlContainer = container;
+        pulsarCluster.startService(DebeziumMsSqlContainer.NAME, debeziumMsSqlContainer);
+    }
+
+    @SneakyThrows
+    @Override
+    public void prepareSource() {
+        runSqlCmd("CREATE DATABASE TestDB;", false);
+        runSqlCmd("EXEC sys.sp_cdc_enable_db;");
+        ContainerExecResult res = runSqlCmd("SELECT is_cdc_enabled FROM sys.databases WHERE database_id = DB_ID();");
+        // " 1" to differentiate from "(1 rows affected)"
+        Assert.assertTrue(res.getStdout().contains(" 1"));
+        runSqlCmd("CREATE TABLE customers (" +
+                "id INT NOT NULL  IDENTITY  PRIMARY KEY, " +
+                "first_name VARCHAR(255) NOT NULL, " +
+                "last_name VARCHAR(255) NOT NULL, " +
+                "email VARCHAR(255) NOT NULL" +
+                ");");
+        runSqlCmd("EXEC sys.sp_cdc_enable_table @source_schema = 'dbo', @source_name = 'customers',"
+                + " @role_name = NULL, @supports_net_changes = 0, @capture_instance = 'dbo_customers_v2';");
+        runSqlCmd("EXEC sys.sp_cdc_start_job;");
+    }
+
+    private ContainerExecResult runSqlCmd(String cmd) throws Exception {
+        return runSqlCmd(cmd, true);
+    }
+
+    private ContainerExecResult runSqlCmd(String cmd, boolean useTestDb) throws Exception {
+        log.info("Executing \"{}\"", cmd);
+        ContainerExecResult response = this.debeziumMsSqlContainer
+                .execCmd("/bin/bash", "-c",
+                "/opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P \""
+                        + DebeziumMsSqlContainer.SA_PASSWORD + "\" -Q \""
+                        + (useTestDb ? "USE TestDB; " : "")
+                        + cmd
+                        + "\""
+                );
+        if (Strings.isNullOrEmpty(response.getStderr())) {
+            log.info("Result of \"{}\":\n{}", cmd, response.getStdout());
+        } else {
+            log.warn("Result of \"{}\":\n{}\n{}", cmd, response.getStdout(), response.getStderr());
+        }
+        return response;
+    }
+
+    @Override
+    public void prepareInsertEvent() throws Exception {
+        runSqlCmd("INSERT INTO customers (first_name, last_name, email) VALUES ('John', 'Doe', 'jdoe@null.dev');");
+        runSqlCmd("SELECT * FROM customers WHERE last_name='Doe';");
+    }
+
+    @Override
+    public void prepareDeleteEvent() throws Exception {
+        runSqlCmd("DELETE FROM customers WHERE last_name='Doe';");
+        runSqlCmd("SELECT * FROM customers WHERE last_name='Doe';");
```

Comment:
why do you need to run this SELECT statement ?

Reply:
This is to simplify troubleshooting later (so the result makes it to the log)